sources/input_to_text_tree.c

```diff
@@ -3,14 +3,15 @@
 /*                                                        :::      ::::::::   */
 /*   input_to_text_tree.c                               :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/09/14 23:07:09 by inikulin          #+#    #+#             */
-/*   Updated: 2024/09/29 23:28:43 by inikulin         ###   ########.fr       */
+/*   Updated: 2024/10/31 13:44:08 by taretiuk         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
+#include "parsing/input_processing.h"
```

Comment:
please place this include in the second section, without the mock

Reply:
i deleted it totally, no need